src/SortPlayers.java

```diff
@@ -0,0 +1,43 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class SortPlayers {
+
+    public void sortByNameAscending(List players) {
```

Comment:
Add list content type

Reply:
I thought would be better if there is option to add any list type.

Reply:
That's not how it works because code in this method actually relies on the fact that contect of the list is Player if it would be anything else Employee for example it would fail. That's why you should add a type.

Reply:
ok is added.